rclcpp/include/rclcpp/executor.hpp

```diff
@@ -323,6 +328,9 @@ class Executor
    * single-threaded model of execution.
    * Adding subscriptions, timers, services, etc. with blocking callbacks will cause this function
    * to block (which may have unintended consequences).
+   *
+   * It tries to collect and execute work reapeatedly that are available in any entitites within
+   * a duration or until no more work is available in any entities.
```

Comment:
I don't think this is necessary since this is essentially what the brief description says, no?